tests/template_tests/filter_tests/test_force_escape.py

```diff
@@ -46,11 +48,13 @@ def test_force_escape06(self):
         output = self.engine.render_to_string('force-escape06', {"a": "x&y"})
         self.assertEqual(output, "x&amp;y")
 
+    @ignore_warnings(category=RemovedInDjango20Warning)
     @setup({'force-escape07': '{% autoescape off %}{{ a|escape|force_escape }}{% endautoescape %}'})
     def test_force_escape07(self):
         output = self.engine.render_to_string('force-escape07', {"a": "x&y"})
         self.assertEqual(output, "x&amp;y")
```

Comment:
Ha! So it's actually related to your changes :-) Happy to hear that we'll eventually get the semantics that I would expect.